roles/istio/tasks/certificates.yaml

```diff
@@ -0,0 +1,144 @@
+---
+
+################### Start Get Istio External IP ###################
+
+#- name: Wait for External IP Address to be ready
+#  community.kubernetes.k8s_info:
+#    api_version: v1
+#    kind: Service
+#    name: istio-ingressgateway
+#    namespace: istio-system
+#    wait: yes
+#    wait_timeout: 360
+#  register: istio_web_service
+#  until: istio_web_service.resources[0].status.loadBalancer.ingress[0].ip != ""
+#  retries: 5
+#  delay: 2
+
+
+- name: Set fact about Istio Ingress Gateway External IP
+  set_fact:
+    istio_external_ip: "35.123.14.14141" #"{{ istio_web_service.resources[0].status.loadBalancer.ingress[0].ip }}"
```

Comment:
I forgot to undo this bit (was testing creating certs without istio) while I was doing the real test.

It looks like this info isn't strictly required. Need to investigate why

Reply:
That looks worrying. In other roles I have a logic that reads `istio_external_ip` automatically as that information is sometimes required.